manifests/components/blackbox-exporter.jsonnet

```diff
@@ -0,0 +1,107 @@
+// vim set syntax=jsonnet
+// Imports
+local kube = import "../vendor/github.com/bitnami-labs/kube-libsonnet/kube.libsonnet";
+local kubecfg = import "kubecfg.libsonnet";
+
+// Cluster-specific configuration
+local BLACKBOX_EXPORTER_IMAGE = "bitnami/blackbox-exporter:0.18.0-debian-10-r83";
```

Comment:
This should be in `manifests/components/images.json`, as that file is automatically updated by our CI/CD system.

Reply:
I see, I'll change this.

Reply:
done.